truss/util/path.py

```diff
@@ -84,6 +84,14 @@ def build_truss_target_directory(stub: str) -> Path:
     return target_directory_path
 
 
+def load_trussignore_patterns_from_truss_dir(truss_dir: Path) -> List[str]:
+    truss_ignore_file = truss_dir / ".truss_ignore"
```

Comment:
nit: can we use the same constant that we use for the main .truss_ignore file

Reply:
was able to unify the uses of this filename within the `path.py` file and created a constant